exporters/requirements.txt

```diff
@@ -3,6 +3,7 @@ openshift == 0.13.1    # module openshift
 prometheus_client      # module prometheus_client
 attrs                  # module attrs
 exceptiongroup         # module exceptiongroup
+typing-extensions      # module typing_extensions
```

Comment:
what you think of using [Poetry](https://python-poetry.org/) (or other similar tool) for managing the project's dependencies (and fixing the versions)? (but better as separate PR)

Reply:
The last time I checked, `poetry` is not supported by the `ubi8-python` image we use.

We also decided a while back that we _wanted_ to have dependencies upgrade on their own, to make sure we can keep up with security releases without manual intervention.

I also have other criticisms of poetry, but that's a discussion for another time.

Reply:
We should discuss this at sync. And we should document this in our architecture docs!